tools/lsp/server_loop.rs

```diff
@@ -338,6 +347,80 @@ pub fn query_properties_command(
     }
 }
 
+pub fn set_binding_command(
```

Comment:
Should this function not be moved to the `properties` module?
I think it belong there, with all the other stuff related to the property things.

Reply:
Fair point.
I see the modules as a way to sort the code such that it makes it easier to find where the code is.
In that respect, it feels like everything releated to something should go in that module, because if it's about that you would first look in that module.
The server_loop use to have everything, but then when parts became too big, such as completion, or goto symbol, everything was moved to that module.

Reply:
Ok.
( but the decoding/encoding for the completion thing is in the completion module. That said, for the preview it isn't)